app/Resources/views/autoEdits/result.html.twig

```diff
@@ -0,0 +1,68 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+<div class="col-lg-12">
+<div id="wrapper">
+<main id="content" class="clearfix">
+        <div class="col-lg-8 col-lg-offset-2">
+            </div>
```

Comment:
I don't think you need any of the above elements, since they are in base.html.twig